src/vm/threads.h

```diff
@@ -2878,9 +2878,7 @@ class Thread: public IUnknown
         STR_SwitchedOut,
     };
 
-#if defined(FEATURE_HIJACK) && defined(PLATFORM_UNIX)
```

Comment:
Why was this `#ifdef` removed? The method is only for FEATURE_HIJACK && PLATFORM_UNIX. The definition of the method is under the same `#ifdef`.